lib/howl/commands/app_commands.moon

```diff
@@ -665,39 +665,107 @@ do_search = (search, whole_word) ->
       m = matches[i]
       file_search_hit_to_location(m, search, display_as)
 
-  locations, searcher, project
+  locations, searcher
 
-command.register
-  name: 'project-file-search',
-  description: 'Searches files in the the current project'
-  input: (opts) ->
-    editor = app.editor
-    search = nil
-    whole_word = false
+ask_for_search_directory = (search_query) ->
+  buffer = app.editor.buffer
+  file = buffer.file or buffer.directory
+  project = Project.for_file(file) if file
 
-    if opts.text and not opts.text.is_empty
-      search = opts.text
+  opts =
+    title: if search_query
+      "(Please specify a directory to search for '#{search_query}'): "
+    else
+      "(Please specify a directory to search): "
```

Comment:
Looks like we ensure `search_query` exists and is non-empty before calling this function - so this `else` is unnecessary?

Reply:
Done